internal/config/config.go

```diff
@@ -0,0 +1,111 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"gopkg.in/yaml.v3"
+)
+
+// ArtiCtlConfig is a config for artictl.
+type ArtiCtlConfig struct {
+	CurrentContext string                 `yaml:"currentContext"`
+	Contexts       map[string]*CtlContext `yaml:"contexts"`
+}
+
+// CtlContext represents credentials for access to the Artipie server.
+type CtlContext struct {
+	Auth     Auth   `yaml:"-"`
+	Endpoint string `yaml:"endpoint"`
+}
+
+type errConfigNotFound struct{ paths []string }
+
+func (e *errConfigNotFound) Error() string {
+	return fmt.Sprintf("file not found, paths: %v",
+		strings.Join(e.paths, ", "))
+}
+
+type errConfigFieldEmpty struct{ field string }
```

Comment:
And same here: we may want to check this error later:
```suggestion
type ErrConfigFieldEmpty struct{ Field string }
```